changelogs/unreleased/367-id-attribute-called-id.yml

```
@@ -0,0 +1,7 @@
+description: In a future release it will not be possible anymore to use an id_attribute called id
```

Just to add a bit more context.
```suggestion
description: In a future release it will not be possible anymore to use a resource with an id_attribute called id
```